### ISSUE TITLE
CUDA kernel not found in registries for Op type: Pow 

### ISSUE THREAD

**Issue by II245** [microsoft/onnxruntime#7097]
**Describe the bug**
When I export a model from Pytorch (1.8.0) to onnx in fp16 mode I get warnings at runtime. Please, see below.

```
1.8.0
{'o': tensor([2.1899e-01, 9.2920e-01, 4.8280e-05, 7.2607e-01, 1.1436e-02, 7.0605e-01,
        4.9622e-02, 1.7566e-01, 6.3867e-01, 4.8755e-01], device='cuda:0',
       dtype=torch.float16)}
2021-03-22 14:44:36.656817045 [W:onnxruntime:Default, cuda_execution_provider.cc:1885 GetCapability] CUDA kernel not found in registries for Op type: Pow node name: Pow_1
[array([2.190e-01, 9.292e-01, 4.828e-05, 7.261e-01, 1.144e-02, 7.061e-01,
       4.962e-02, 1.757e-01, 6.387e-01, 4.875e-01], dtype=float16)]
```

**Urgency**
none.

**System information**
 - OS Platform and Distribution: Linux Ubuntu 18.04: Linux x86_64
 - ONNX Runtime installed from (source or binary): binary
 - ONNX Runtime version: onnxruntime-1.6.0
 - Python version: Python 3.8.5
 - Pytorch version: 1.8.0
 - CUDA/cuDNN version: CUDA Version: 10.2
 - GPU model and memory: Tesla V100 (32G)

**To Reproduce**
```python
import onnxruntime
from torch import nn
import torch

print(torch.__version__)


class Model(nn.Module):
	def forward(self, x):
		x = x.pow(2)
		return dict(o=x)


model = Model()
model.to(dtype=torch.float16, device='cuda')
x = torch.rand(10).to(dtype=torch.float16, device='cuda')

print(model(x))

torch.onnx.export(
		model, (x,),
		'fp16_pow_repro.onnx',
		verbose=False,
		opset_version=12,
		export_params=None,
		do_constant_folding=True,
		input_names=['x']
)

runtime = onnxruntime.InferenceSession('fp16_pow_repro.onnx')
print(runtime.run(None, dict(x=x.cpu().numpy())))
```
[fp16_pow_repro.zip](https://github.com/microsoft/onnxruntime/files/6185592/fp16_pow_repro.zip)

**Expected behavior**
Inference without warnings.

Thank you for your reply!

**Comment by skottmckay** [comment]
The fp16 Pow kernel was added after the 1.6 release. It's available in the 1.7 release.

The warning is meaningful as performance will degrade when we have to switch between CUDA and CPU to execute a node.

**Comment by II245** [comment]
Thanks for reply!
